src/chainparams.cpp

```diff
@@ -298,3 +320,24 @@ bool SelectParamsFromCommandLine()
     SelectParams(network);
     return true;
 }
+
+
+// Block height must be >0 and less than the halving interval block height
+// Index variable i ranges from 0 - (vFoundersRewardAddress.size()-1)
+std::string CChainParams::GetFoundersRewardAddress(size_t nHeight) const
+{
+    assert(nHeight>0 && nHeight<consensus.nSubsidyHalvingInterval);
+    size_t i = (size_t)floor((double(nHeight)/consensus.nSubsidyHalvingInterval)*vFoundersRewardAddress.size());
```

Comment:
Please check the proposed specification at https://github.com/zcash/zips/blob/zips63.specify-founders-reward.0/protocol/protocol.pdf (sections 6.5 and 6.6).